### PR TITLE
new module gridss/gridss

### LINKED ISSUE #2879: new module: gridss/gridss
### Is there an existing module for this?

- [X] I have searched for the existing module

### Is there an open PR for this?

- [X] I have searched for existing PRs

### Is there an open issue for this?

- [X] I have searched for existing issues

### Are you going to work on this?

- [X] If I'm planning to work on this module, I added myself to the `Assignees` to facilitate tracking who is working on the module

### PR DESCRIPTION
**Pull request by nvnieuwk**
## PR checklist

Closes #2879 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Review by matthdsm — APPROVED** [review]
LGTM! (But I don't count 😢 )

**Comment by nvnieuwk** [comment]
It's a new module, that won't break anything :p

**Comment by matthdsm** [comment]
Just following @maxulysse 's orders xD

**Comment by maxulysse** [comment]
> Just following @maxulysse 's orders xD

What now?

**Comment by matthdsm** [comment]
> > Just following @maxulysse 's orders xD
> 
> What now?

You said we couldn't approve each others PR's anymore 🤷🏻

**Review by peterpru — APPROVED** [review]
Am I allowed to approve these changes? Looks good to me. :)

File: modules/nf-core/gridss/gridss/main.nf [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        ${assembly} \\
+        --jvmheap ${task.memory.toGiga()}g \\
+        --otherjvmheap ${task.memory.toGiga() - 1}g \\
+        ${inputs}
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        gridss: ${VERSION}
+    END_VERSIONS
+    """
```
Comment (maxulysse) [inline modules/nf-core/gridss/gridss/main.nf:48]: Can we get some `stub`

Comment (nvnieuwk) [reply modules/nf-core/gridss/gridss/main.nf:48]: Oops forgot to add it 😁

Comment (matthdsm) [reply modules/nf-core/gridss/gridss/main.nf:48]: Is the `stub` section a new requirement? (good to know for future reviews)

**Pull request merged**